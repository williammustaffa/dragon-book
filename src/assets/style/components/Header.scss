#header {
  color: $color-dark-black;

  .header {
    &-logo-container {
      display: table;
      margin: $spacing * 2 auto;
      text-align: center;
    }

    &-logo {
      display: inline-block;
      max-width: 100px;
    }

    &-logo-title {
      margin-top: $spacing;
      text-transform: uppercase;
      font-size: $font-size-big;
    }
  }
}