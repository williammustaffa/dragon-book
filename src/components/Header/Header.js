import React from "react";
import { Image } from "semantic-ui-react";
import logo from "assets/images/logo.png";

function Header() {
  return (
    <div id="header">
      <div className="header-logo-container">
        <Image src={logo} className="header-logo" />
        <div className="header-logo-title">
          Dragons Book
        </div>
      </div>
    </div>
  );
}

export default Header;