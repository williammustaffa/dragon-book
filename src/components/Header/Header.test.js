import React from "react";
import Header from "./Header";
import { render } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"

test("Test Header component", () => {
  const { asFragment } = render(<Header />);

  // Run tests
  expect(asFragment()).toMatchSnapshot();
});