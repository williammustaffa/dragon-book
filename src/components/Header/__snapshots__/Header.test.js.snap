// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header component 1`] = `
<DocumentFragment>
  <div
    id="header"
  >
    <div
      class="header-logo-container"
    >
      <img
        class="ui image header-logo"
        src="logo.png"
      />
      <div
        class="header-logo-title"
      >
        Dragons Book
      </div>
    </div>
  </div>
</DocumentFragment>
`;
